scheduler/src/cook/mesos/scheduler.clj

```diff
@@ -47,12 +51,18 @@
   []
   (java.util.Date.))
 
+(defn offer-resource-value
+  ([offer resource-name]
+   (offer-resource-value offer resource-name :scalar))
+  ([offer resource-name value-type]
+   (->> offer :resources (filter #(= (:name %) resource-name)) first value-type)))
```

Comment:
Hmm, that's a good point.  I didn't consider multiple roles for the same resource in the same offer....  Since Cook can only run with 1 role, and mesos is aware of the role, we can assume that any resource items will either be for Cook's role or for "*" right?   And therefore, for Cook's purposes, we can simply sum the values for each resource entry of the appropriate type... correct?